examples/Choropleth.ipynb

```diff
@@ -51,6 +52,45 @@
     "m"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# To add callback for style\n",
+    "def compute_style(feature, colormap, choro_data):\n",
+    "    return {\n",
+    "        'fillColor': colormap(choro_data),\n",
+    "        'weight': random.randint(1, 5),\n",
+    "    }"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "layer_callback = ipyleaflet.Choropleth(\n",
+    "    geo_data=geo_json_data,\n",
+    "    choro_data=unemployment,\n",
+    "    colormap=linear.YlOrRd_04, \n",
+    "    border_color='black',\n",
+    "    style_callback=compute_style\n",
+    ")"
```

Comment:
I meant here you can replace the entire code by: 
```python
layer.style_callback = compute_style
```

Reply:
Done :)